xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/AttachmentsPane.java

```diff
@@ -188,7 +189,10 @@ public void deleteAttachmentByFileByName(String attachmentName)
             + "']/parent::div/following-sibling::div[contains(@class, 'deleteAttachment')]"));
         WebElement deleteButton = this.attachmentsLivetable.getCell(getRowIndexByAttachmentName(attachmentName), 6)
             .findElement(By.className("actiondelete"));
-        deleteButton.click();
+        // This is needed since Selenium deleteButton.click() fails on smaller windows, even if the element is
+        // clickable (visible and enabled). The failure was reproduced on {@code org.xwiki.test.ui.CompareVersionsTest}
```

Comment:
`@code` is only for javadoc and is not needed for inline comments.

Reply:
Removed:)